content/guides/output-management.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Output Management
```

Comment:
Maybe it's just me, but I have some troubles guessing the content from this title. "Building index.html" or "Generating index.html" disregards the last two sections, but might be easier to grasp. Any thoughts?